corehq/apps/reports/static/reports/js/bootstrap3/application_status.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+hqDefine("reports/js/bootstrap3/application_status", [
+    "jquery",
+], function (
+    $
+) {
+    $(function () {
+        $('#report-content').on('click', '.toggle-all-locations', function (e) {
+            $(this).prevAll('.locations-list').toggle();
+            $(this).children('span').toggle();
```

Comment:
nit: would be nice to limit this by adding a specific class to this span to avoid conflicting with any other span getting added to the div in future.

Reply:
[d5b5728](https://github.com/dimagi/commcare-hq/pull/34744/commits/d5b572806cbe8fc4e6c992741a24a1c99a21aac2)